charts/kafka/README.md

```diff
@@ -1,25 +1,11 @@
 <!---
-Copyright 2020 Crown Copyright
```

Comment:
My previous comment about contradictory was the way we have two mentions of where we got this file from which point to two different addresses. We will need this licence header putting back in